### DISCUSSION TITLE
Planet scale free tier

### DISCUSSION THREAD

**Discussion by alaidine** [planetscale/discussion#630] [General]
(Sorry for my bad english, it is not my first language) I am a student in student in Reunion Island. I had a free tier subscription on planetscale for a side project. When it was deprecated planet scale took money from my bank account even though i didnt upgrade my plan. Is it possible to get a refund because I need that money to eat ?

**Comment by mscoutermarsh** [comment]
Hi @alaidine,

Could you please contact support? https://planetscale.com/contact
